server/handlers/handlers.go

```diff
@@ -186,7 +188,7 @@ func (ah *ActivityHandler) handleActivity(activity msteams.Activity) {
 	switch activity.ChangeType {
 	case "created":
 		ah.plugin.GetAPI().LogDebug("Handling create activity", "activity", activity)
-		discardedReason = ah.handleCreatedActivity(activityIds)
+		discardedReason = ah.handleCreatedActivity(activityIds, activity.ActivityReceiveTime)
```

Comment:
We are doing this only on creation, should we do it also on update and delete?

Reply:
Maybe with a `action_type` label?

Reply:
I'd be ok starting with just create so we can evaluate if this is a useful metric. The primary goal was to measure the "roundtrip time" in both directions, and while we could technically do that for all change events, it perhaps suffices to just track created posts for now as the logic is probably simpler.